articles/atom-vs-visual-studio-code-functionality-and-features-comparison/index.md

```diff
@@ -0,0 +1,178 @@
+
+VS Code is an open-source code editor developed by Microsoft in April 2015. There is no payment required to use VS Code Editor.
+
+It uses Electron as its base, thus enabling cross-platform. It works on Windows, Mac OS, and Linux. Electron is built using Node.js, and it has built-in support for JavaScript and TypeScript. VS Code is built using web technology: HTML, CSS, and JavaScript.
+
+Visual Studio Code supports other languages such as Java, Python, PHP, C#, and C++. It comprises thousands of extensions that enable debuggers, more languages, commands, themes, etc.
+
+#### How to Use Visio Studio Code Editor
+
+To get started, download VS Code editor from [Visual Studio home page](https://code.visualstudio.com/). 
+
+![VS Code download](/engineering-education/atom-vs-visual-studio-code-functionality-and-features-comparison/vsdownload.PNG)
+
+Go to the site and download the latest version of the editor. Currently, it is at V1.51. After installing the editor, you will see the welcome screen guide.
+
+![VS Code Welcome Guide](/engineering-education/atom-vs-visual-studio-code-functionality-and-features-comparison/vswelcome.PNG)
+
+The activity bar on the left with icons give you access to:
+
+- Explorer
+- Search
+- Source Control
+- Debugger
+- Extensions
+
+At the explorer, you go through the project files and where code writing and editing takes place. The search icon is used while finding files and information. Source Control Icon allows you to initialize the Git repository and manage projects. At the extension's icon, you install can install and manage extensions. VS Code comes with many integrated built-in features. You can learn more about Visual Studio editor [here.](https://code.visualstudio.com/docs)
+
+### Atom vs. VS Code: Tabular Comparison
+
+Let's have a glance at features offered by Atom and VS Code text editors.
+
+| Category | Atom | Visual Studio Code |
+| --- | --- | --- |
+| Released on | February 2014 | April 2015 |
+| Developers | GitHub | Microsoft |
+| Supported by | Windows / Linux / Mac OS | Windows / Linux / Mac OS |
+| License | MIT License | MIT License |
+| Extension Support | Yes | Yes |
+| Cross-platform support | Yes | Yes |
+| Syntax highlighting | Yes | Yes |
+| Auto-completion | Yes | Yes |
+| Inbuilt Version Control | Yes | Yes |
+| Multiple selection editing | Yes | Yes |
+| Price | Free | Free |
+| Find and replace | Yes | Yes |
+
+### Atom vs. VS Code: Features and Functionality Comparison
+
+1. Configuration
+
+With loads of extensions and customization comes many configurations. The configuration of applications is fundamental. It determines the overall user experience with the editor.
+
+Configuration in Visual Studio Code involves a JSON file recently replaced by Graphical User Interface (GUI). It is simpler to use GUI, and it does its work well. The level of abstraction is maintained while using GUI over JSON.
```

Comment:
> Configuration in Visual Studio Code involves a JSON file recently replaced by Graphical User Interface (GUI). It is simpler to use GUI, and it does its work well. The level of abstraction is maintained while using GUI over JSON.

Can you illustrate how configuration using a GUI and JSON files work in VS Code? It would help readers understand the pros and cons of using GUI vs JSON.

Please illustrate it with examples.

Reply:
It would be easier for the reader to follow if you could include a hands-on example about this. @JudyNduati